server/index.js

```diff
@@ -14,42 +14,72 @@ const port = 9000;
 app.use(cors());
 app.use(bodyParser.json());
 app.use(express.static('client/dist'));
+app.use('/:id', express.static('client/dist'));
```

Comment:
I don't know if you need the /:id route here. Static assets are for endpoints that don't have additional parameters like :id

Reply:
The first api get-all doesn't have :id param pass in.